DailySelfie/app/src/main/java/com/teamagam/dailyselfie/PictureAdapter.java

```diff
@@ -0,0 +1,52 @@
+package com.teamagam.dailyselfie;
+
+import android.content.Context;
+import android.graphics.BitmapFactory;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import java.util.Collections;
+import java.util.List;
+
+class PictureAdapter extends RecyclerView.Adapter<PictureAdapter.PictureViewHolder> {
+    private LayoutInflater mInflater;
+    private List<PictureInfo> mPictureInfoList = Collections.emptyList();
+
+    PictureAdapter(Context context, List<PictureInfo> pictureInfoList) {
+        mInflater = LayoutInflater.from(context);
+        mPictureInfoList = pictureInfoList;
+    }
+
+    @Override
+    public PictureViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View view = mInflater.inflate(R.layout.layout_picture, parent, false);
```

Comment:
I don't know if there's any difference, but you can get an inflater using:
LayoutInflater.from(parent.getContext())
Just FYI